extensions/src/Smithy.Identity.Abstractions/Smithy.Identity.Abstractions.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Smithy.Identity.Abstractions</AssemblyName>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
```

Comment:
I don't see an `AssemblyInfo.cs` for this project where you would specify all of these things that you have turned off generation. Either add one or remove these and add the required information. I'm not sure how you will handle the assembly versioning strategy we use in the SDK with `AssemblyVersion` and `AssemblyFileVersion` purely with the csproj file so you might need to add an `AssemblyInfo.cs`.

Reply:
Added the assembly info file in https://github.com/aws/aws-sdk-net/pull/3330/commits/285e421c304ad6d5c21deb54e6f85ec374226275#diff-37d2151f0ebc200ab67497b246f32ca3d73c3a9d1a1ac999098f5229eb7ebdcf